rflx/templates/rflx_builtin_types-conversions.ads

```diff
@@ -34,4 +34,30 @@ is
      Pre =>
        Valid_Boolean (Val);
 
+   function Valid_Boolean (Val : {prefix}RFLX_Arithmetic.S63) return Boolean is
+     (case Val is
+         when 0 | 1 =>
+            True,
+         when others =>
+            False);
+
+   function To_Base_Int (Enum : Boolean) return {prefix}RFLX_Arithmetic.S63 is
```

Comment:
I think it can be confusing that the function `To_Base_Int` has a return type `S63`, while usually its subtype `Base_Integer` is used in other places. How about making the naming consistent: `Base_Integer` and `To_Base_Integer`, and replacing `S63` by `Base_Integer` altogether?

Reply:
done